https://scans.gradle.com/s/prj53ehrw5zoi/tests/task/:plugin-gradle:test/details/com.diffplug.gradle.spotless.KotlinExtensionTest/withCustomRuleSetApply()?top-execution=1

```
> Configure project : |  
-- | --
  | e: /tmp/junit16565205775692441507/build.gradle.kts:13:79: Expecting an argument |  
  | e: /tmp/junit16565205775692441507/build.gradle.kts:9:10: Cannot access 'BaseKotlinExtension': it is package-private in 'com.diffplug.gradle.spotless' |  
  | e: /tmp/junit16565205775692441507/build.gradle.kts:13:13: Unresolved reference: ktlint_function_naming_ignore_when_annotated_with |  
  |   |  
  | FAILURE: Build failed with an exception. |  
  |   |  
  | * Where: |  
  | Build file '/tmp/junit16565205775692441507/build.gradle.kts' line: 13 |  
  |   |  
  | * What went wrong: |  
  | Script compilation errors: |  
  |   |  
  | Line 13:             ktlint_function_naming_ignore_when_annotated_with to "Composable", |  
  | ^ Expecting an argument |  
  |   |  
  | Line 09:         .customRuleSets(listOf( |  
  | ^ Cannot access 'BaseKotlinExtension': it is package-private in 'com.diffplug.gradle.spotless' |  
  |   |  
  | Line 13:             ktlint_function_naming_ignore_when_annotated_with to "Composable", |  
  | ^ Unresolved reference: ktlint_function_naming_ignore_when_annotated_with |  
  |   |  
  | 3 errors
```

--- a/plugin-gradle/src/test/java/com/diffplug/gradle/spotless/KotlinExtensionTest.java
+++ b/plugin-gradle/src/test/java/com/diffplug/gradle/spotless/KotlinExtensionTest.java
@@ -156,7 +156,7 @@ class KotlinExtensionTest extends GradleIntegrationHarness {
 				"            \"io.nlopez.compose.rules:ktlint:0.3.3\"",
 				"        ))",
 				"        .editorConfigOverride(mapOf(",
-				"            ktlint_function_naming_ignore_when_annotated_with to \"Composable\",",
+				"            \"ktlint_function_naming_ignore_when_annotated_with\" to \"Composable\",",
 				"        ))",
 				"    }",
 				"}");

--- a/plugin-gradle/src/test/java/com/diffplug/gradle/spotless/KotlinExtensionTest.java
+++ b/plugin-gradle/src/test/java/com/diffplug/gradle/spotless/KotlinExtensionTest.java
@@ -156,7 +156,7 @@ class KotlinExtensionTest extends GradleIntegrationHarness {
 				"            \"io.nlopez.compose.rules:ktlint:0.3.3\"",
 				"        ))",
 				"        .editorConfigOverride(mapOf(",
-				"            \"ktlint_function_naming_ignore_when_annotated_with\" to \"Composable\",",
+				"            \"ktlint_function_naming_ignore_when_annotated_with\" to \"Composable\"",
 				"        ))",
 				"    }",
 				"}");

